review-tools/ghmerge

```diff
@@ -82,14 +88,19 @@ function cleanup {
 }
 trap 'cleanup' EXIT
 
-git pull --rebase https://github.com/$WHO/$WHAT.git $BRANCH
+echo appending $REPO/$BRANCH
+git pull --rebase $REPO $BRANCH
+echo rebasing on $REL
 git rebase $REL
 
 echo Diff against $REL
 git diff $REL
 
-echo -n Press return to merge to $REL: ; read foo
 addrev $TRIVIAL --prnum=$PRNUM $TEAM ${REL}..
+echo Log since $REL
+git log $REL..
+
+echo -n Press return to merge to $REL: ; read foo
 git checkout $REL
 if [ "$MERGE" == "yes" ] ; then
     git merge --no-commit --squash $WORK
```

Comment:
Not directly related this, but I'd like to have something more akin to `git rebase -i --autosquash` here.  Being able the tune the squashing is the cause of the majority of times I don't use `ghmerge`.

Reply:
Good point. 
Since you already approved this PR, I suggest you do this in a separate PR.
If you prefer, I can `s/git merge --no-commit --squash/git rebase -i --autosquash/` here.

Reply:
Now I see that `-i --autosquash` should better be added to the existing `git rebase` variant.
I've done this and updated and slightly extended the README accordingly.

Reply:
Thanks, here is best since there are a number of other changes underway.
I'll clear my review.